Feature/sig 1021 location gebieden

JKoetsier opened the pull request.

vanbuiten commented on api/app/signals/apps/signals/migrations/0001_initial.py:
```diff
@@ -23,11 +23,25 @@ class Migration(migrations.Migration):
                 'db_table': 'buurt_simple',
             },
         ),
+        migrations.CreateModel(
```
Let's do this as discussed with the `managed=False` on the model

vanbuiten commented on api/app/signals/apps/signals/models.py:
```diff
@@ -35,11 +35,23 @@ class Buurt(models.Model):
+    wkb_geometry = models.PolygonField(srid=4326, null=True)
+
+    class Meta:
+        db_table = 'stadsdeel'
```
Use `managed=False`

JKoetsier commented:
Wait with merging. Need to setup import for gebieden on environments first.

JKoetsier commented:
Reverted managed=False and added models to the initial migration again. Models where managed=False are not created when running the tests. Working around this is hacky in any way. Cleaner and also safer to just leave the model definitions in the initial migration.

vanbuiten commented:
@tcoenen I updated this branch with the latest release
